cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -22,18 +22,18 @@ This comes at the cost of additional storage space and slower writes, so decidin
 Once an index has been created, it will be managed and kept up to date by the DBMS.
 Neo4j will automatically pick up and start using the index once it has been created and brought online.
 
-There are multiple index types available: b-tree (deprecated), fulltext, lookup, and text index types.
+There are multiple index types available: b-tree (deprecated), fulltext, lookup, point, range, and text index types.
```

Comment:
```suggestion
There are multiple index types available:

* Fulltext index.
* Lookup index.
* Point index.
* Range index.
* Text index types.
* B-tree. label:deprecated[]

[NOTE]
====
The deprecated B-tree index type has been replaced by more specific index types (Lookup, Point, and Range).
====
```

I dont know what `Text index types` are..

Reply:
It is text and not lookup that replaces btree, lookup replaces the label store I think. text repalces the `lucene+native-3.0` btree provider.

Reply:
```suggestion
There are multiple index types available:

* Fulltext index.
* Lookup index.
* Point index.
* Range index.
* Text index types.
* B-tree. label:deprecated[]

[NOTE]
====
The deprecated B-tree index type has been replaced by more specific index types (Range, Point, and Text).
====
```